content/about/security.fr.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Security
```

Comment:
Just a quick typo: title should be `security` and not `Security`, I know it's minimal, but it's just to guarantee, your file metadata matches the english one!